src/main/daos/Challenge.kt

```diff
@@ -218,9 +218,9 @@ object SubChallenges : BaseIntIdTable("sub_challenge") {
 }
 
 data class ChallengeNamespace(
-    val parentChallenge: Int?,
+    val parentChallenge: Int? = null,
```

Comment:
suggest moving all non-required params to end -- and in order of how likely it is to be used